src/store/ChartStore.js

```diff
@@ -299,6 +300,7 @@ class ChartStore {
                 }
 
                 this.context = context;
+                this.loader.setState('market-symbol');
```

Comment:
@shayanbinary I dont think its the right place to change symbol, try to change the symbol form your test link and you'll see the `Retrieving Chart Data...`

Reply:
@MahboobehMohammadi whenever we change the symbol, new data for active symbol should be fetch from server, so I think that would be typical